client/src/templates/Challenges/rechallenge/transformers.js

```diff
@@ -54,11 +55,11 @@ async function loadBabel() {
   /* eslint-enable no-inline-comments */
   Babel.registerPlugin(
     'loopProtection',
-    protect(protectTimeout, loopProtectCB)
+    protect(protectTimeout, loopProtectCB, loopsPerTimeoutCheck)
   );
   Babel.registerPlugin(
     'testLoopProtection',
-    protect(testProtectTimeout, testLoopProtectCB, loopsPerTimeoutCheck)
+    protect(testProtectTimeout, testLoopProtectCB, testLoopsPerTimeoutCheck)
```

Comment:
I am sure I have asked this before, but one more time for my sanity: What is the difference between the `loopProtection` and `testLoopProtection`?

Reply:
Just the configuration. The idea is that `testLoopProtection` is less aggressive (it times out later and checks the timeout less often) as a compromise between protection and allowing slow tests to complete. 

`loopProtection` is more aggressive because people often write infinite loops on the way to writing finite ones, so we don't want people's browsers crashing just because they haven't yet written the code that modifies the conditional, say.